pypsa/standard_types/line_types.csv

```diff
@@ -30,3 +30,4 @@ NAYY 4x50 SE,50.,0.642,0.083,210,0.142,cs,50,pandapower;Heuck et al. Elektrische
 Al/St 240/40 2-bundle 220.0,50.,0.06,0.301,12.5,1.29,ol,240,"Oeding and Oswald ""Elektrische Kraftwerke und Netze"", 2011, Chapter 9"
 Al/St 240/40 3-bundle 300.0,50.,0.04,0.265,13.2,1.935,ol,240,"Oeding and Oswald ""Elektrische Kraftwerke und Netze"", 2011, Chapter 9"
 Al/St 240/40 4-bundle 380.0,50.,0.03,0.246,13.8,2.58,ol,240,"Oeding and Oswald ""Elektrische Kraftwerke und Netze"", 2011, Chapter 9"
+Al/St 560/50 4-bundle 750.0,50.,0.013,0.276,13.13,4.16,ol,560,"Oeding and Oswald ""Elektrische Kraftwerke und Netze"", 2011, Appendix",,,
```

Comment:
```suggestion
Al/St 560/50 4-bundle 750.0,50.,0.013,0.276,13.13,4.16,ol,560,"Oeding and Oswald ""Elektrische Kraftwerke und Netze"", 2011, Appendix"
```

Reply:
Hi Fabian, thanks for changing. Was just about to push the fixed file :D